src/main/java/cuchaz/enigma/gui/panels/PanelEditor.java

```diff
@@ -135,6 +146,19 @@ public void keyReleased(KeyEvent event) {
 		});
 	}
 
+	public void setEditorZoom(Integer zoomAmount) {
```

Comment:
It doesn't really set the zoom level but offsets it. Also, this should be using `int` as well.